core/modules/wdb/QueryRunner.cc

```diff
@@ -441,7 +441,7 @@ class ChunkResourceRequest {
 };
 
 bool QueryRunner::_dispatchChannel() {
-    int fragNum = task->getQueryFragmentNum();
+    int fragNum = _task->getQueryFragmentNum();
     proto::TaskMsg& m = *_task->msg;
```

Comment:
It looks like all uses of this variable `m` are gone now. Perhaps it's time to get rid of it as well?

Reply:
It's still in use.